### PR TITLE
[DTM] - Documentation changes for CD 1020.1.8

### PR DESCRIPTION
**Pull request by Vagdevi1998**
* User documentation changes for CD 1020.1.8
* Changelogs for CD 1020.1.8

### CODE REVIEW

File: content/dtm/asset-types-bundle/asset-models.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ Once all required asset models are created, you can start creating [assets](/dtm
 4. Click the remove icon <i class="dlt-c8y-icon-minus-circle text-danger icon-20"></i> next to the checkbox to remove the child asset model from the selected asset model.
 
 {{< c8y-admon-info>}}
-If you do not select and check a child asset model, then the asset cannot have any subassets.
+If you do not select and check a child asset model, then the asset cannot have any subassets. <br>
+The selected child asset models are not listed in the dropdown.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-models.md:120]:
```suggestion
The selected child asset models are not listed in the dropdown menu.
```

File: content/change-logs/application-enablement/dtm-1020-1-8-renamed-button-names-in-DTM-sections.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-1308
+version: 1020.1.8
+---
+Users can now see simplified button names in different sections of the DTM interface: 'Add Assets,' 'Import' in the **Assets** page;
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020-1-8-renamed-button-names-in-DTM-sections.md:17]:
```suggestion
Updated action button labels: in the **Assets** page, "Import assets" is now "Import". In the **Asset Models** page, "Import asset models" and "Export asset models" are now "Import" and "Export" respectively. Similarly in the **Asset properties** page, "Import asset properties" and  "Export asset properties" are also shortened to "Import" and "Export" respectively.
```

File: content/dtm/asset-types-bundle/asset-models.md [hunk 3]
```diff
@@ -116,7 +116,8 @@ Once all required asset models are created, you can start creating [assets](/dtm
 4. Click the remove icon <i class="dlt-c8y-icon-minus-circle text-danger icon-20"></i> next to the checkbox to remove the child asset model from the selected asset model.
 
 {{< c8y-admon-info>}}
-If you do not select and check a child asset model, then the asset cannot have any subassets.
+If you do not select and check a child asset model, then the asset cannot have any subassets. <br>
+The selected child asset models are not listed in the dropdown menu.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-models.md:120]: This is understood and I don't think is need to be added in the user doc.

File: content/dtm/asset-types-bundle/asset-models.md [hunk 4]
```diff
@@ -138,7 +139,8 @@ If the asset model is "Rotor" and you select "Blade" as an allowed child asset m
 
 {{< c8y-admon-info>}}
 To assign a location to an asset, add the default property Location to the asset model. For more info on default properties, see [Default properties](#default-properties).<br>
-If there are any validation errors for a field, it is highlighted in red with a validation error message asking you to fill in the required information. Enter the necessary information and click **Save**.
+If there are any validation errors for a field, it is highlighted in red with a validation error message asking you to fill in the required information. Enter the necessary information and click **Save**. <br>
+The selected asset properties are not listed in the dropdown.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-models.md:143]: refer previous comment

File: static/images/dtm/asset-type/dtm-asset-type-import.png
Comment (lra-sag) [inline static/images/dtm/asset-type/dtm-asset-type-import.png:1]: The image looks stretched. Please confirm if it looks right in the preview

File: static/images/dtm/asset-type/dtm-asset-type-view.png
Comment (lra-sag) [inline static/images/dtm/asset-type/dtm-asset-type-view.png:1]: same as the previous one

File: static/images/dtm/assets/dtm-assets-asset-hierarchy.png
Comment (lra-sag) [inline static/images/dtm/assets/dtm-assets-asset-hierarchy.png:1]: Please verify all the images

**Review by lra-sag — DISMISSED** [review]

File: content/change-logs/application-enablement/dtm-1020-1-8-additional-columns-in-preview-step-of-import-asset-models.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-05-30
```
[the remark this answers is not in the document]
Comment (pavanshivajirao) [reply content/change-logs/application-enablement/dtm-1020-1-8-additional-columns-in-preview-step-of-import-asset-models.md:2]: Yes Beate, this is discussed internally and decided to keep them separate always going forward. And yes, deployment date is correct as a few entries are missed from that deployment they are coming a bit late though this.

File: content/change-logs/application-enablement/dtm-1020-1-8-additional-columns-in-preview-step-of-import-asset-models.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-1307
+version: 1020.1.8
+---
+In the **Import asset model** dialog, users can now see Child asset models and Asset properties columns in the preview step.
```
[the remark this answers is not in the document]
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-8-additional-columns-in-preview-step-of-import-asset-models.md]: Yes. The columns are also formatted in bold. Modified accordingly.

File: content/change-logs/application-enablement/dtm-1020-1-8-improved-dropdown-for-child-asset-models-and-properties-page.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-05-30
+title: Improved dropdown for Child Asset Models and Asset properties
```
[the remark this answers is not in the document]
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-8-improved-dropdown-for-child-asset-models-and-properties-page.md]: Modified

File: content/change-logs/application-enablement/dtm-1020-1-8-improved-dropdown-for-child-asset-models-and-properties-page.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-1217
+version: 1020.1.8
+---
+When adding child asset models or asset properties to an asset model, those already added will no longer appear in the options.
```
[the remark this answers is not in the document]
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-8-improved-dropdown-for-child-asset-models-and-properties-page.md]: Modified

File: content/change-logs/application-enablement/dtm-1020-1-8-renamed-labels-for-buttons.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-1308
+version: 1020.1.8
+---
+
+Updated action button labels: in the **Assets** page, "Import assets" is now "Import". In the **Asset Models** page, "Import asset models" and "Export asset models" are now "Import" and "Export" respectively. Similarly in the **Asset properties** page, "Import asset properties" and "Export asset properties" are also shortened to "Import" and "Export" respectively.
```
[the remark this answers is not in the document]
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-8-renamed-labels-for-buttons.md]: Modified as suggested

File: content/change-logs/application-enablement/dtm-1020-1-8-renamed-labels-for-buttons.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+'Add Asset Model,' 'Import,' and 'Export' in the **Asset model** page.
```
[the remark this answers is not in the document]
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-8-renamed-labels-for-buttons.md]: Yes. I have removed those lines and modified as suggested

File: static/images/dtm/assets/dtm-assets-asset-hierarchy.png
Comment (Vagdevi1998) [reply static/images/dtm/assets/dtm-assets-asset-hierarchy.png:1]: Modified

File: static/images/dtm/asset-type/dtm-asset-type-import.png
Comment (Vagdevi1998) [reply static/images/dtm/asset-type/dtm-asset-type-import.png:1]: Modified the screenshots

File: static/images/dtm/asset-type/dtm-asset-type-view.png
Comment (Vagdevi1998) [reply static/images/dtm/asset-type/dtm-asset-type-view.png:1]: Modified

**Comment by MWindel** [comment]
@Vagdevi1998 Regarding the screenshots, please change the following:

- static/images/dtm/custom-property/dtm-property-library-type-enum.png --> Remove the cursor showing in the **Values** field.
- static/images/dtm/custom-property/dtm-property-library-simple-custom-property.png --> Remove the cursor showing in the **Default value** field. Also for the UI, in general we do not capitalize two words (for example, "Asset properties"). This also needs to be fixed and the screenshot retaken.

> static/images/dtm/bulk-import/dtm-bulk-import-import-assets-window.png --> This shows too much empty space below the dialog window. Please crop to size.

**Comment by Vagdevi1998** [comment]
Modified the screenshots as suggested

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
Approved for the change logs part. @MWindel will review and approve the user docs part.

**Review by pavanshivajirao — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**